Pod/Classes/BONChain.h

```diff
@@ -17,8 +17,15 @@ typedef BONChain * (^BONChainFont)(UIFont *font);
 typedef BONChain * (^BONChainColor)(UIColor *color);
 typedef BONChain * (^BONChainAdobeTracking)(NSInteger adobeTracking);
 typedef BONChain * (^BONChainPointTracking)(CGFloat pointTracking);
+typedef BONChain * (^BONChainFirstLineHeadIndent)(CGFloat firstLineHeadIndent);
+typedef BONChain * (^BONChainHeadIndent)(CGFloat headIndent);
+typedef BONChain * (^BONChainTailIndent)(CGFloat tailIndent);
 typedef BONChain * (^BONChainLineHeight)(CGFloat lineHeightMultiple);
+typedef BONChain * (^BONChainMaximumLineHeight)(CGFloat maximumLineHeight);
+typedef BONChain * (^BONChainMinimumLineHeight)(CGFloat minimumLineHeight);
 typedef BONChain * (^BONChainLineSpacing)(CGFloat lineSpacing);
+typedef BONChain * (^BONChainParagraphSpacing)(CGFloat paragraphSpacing);
```

Comment:
Please rename the block typedef to match. (Left this comment in the wrong place before)